src/Rules/Properties/PropertiesInInterfaceRule.php

```diff
@@ -75,6 +75,37 @@ public function processNode(Node $node, Scope $scope): array
 			];
 		}
 
+		if ($node->isAbstract()) {
+			return [
+				RuleErrorBuilder::message('Property in interface cannot be explicitly abstract.')
+					->nonIgnorable()
+					->identifier('property.hookedStatic')
```

Comment:
wrong identifier here

Reply:
Still wrong? `property.hookedStatic` does not make sense for `Property in interface cannot be explicitly abstract.`?

Reply:
argh, sorry